src/components/UserProfile/UserProfile.jsx

```diff
@@ -702,6 +702,7 @@ function UserProfile(props) {
   const canUpdatePassword = props.hasPermission('updatePassword');
   const canGetProjectMembers = props.hasPermission('getProjectMembers');
   const canChangeRehireableStatus = props.hasPermission('changeUserRehireableStatus');
+  const canUpdateSummaryRequirements = props.hasPermission('updateSummaryRequirements');;
```

Comment:
Minor change on the two semicolons, it is likely to break the code.

Reply:
Fixed that. Thanks.